docs/hub/unity-sentis.md

```diff
@@ -0,0 +1,66 @@
+# Using Sentis Models from Hugging Face
+
+[Unity Sentis](https://unity.com/products/sentis) is the inference engine that runs on Unity 2023 or above. 
+
+## Exploring Sentis Models in the Hub
+You will find `unity-sentis` models by filtering at the left of the [models page](https://huggingface.co/models?library=unity-sentis).
+
+All the Sentis models in the Hub come with code and instructions to easily get you started using the model in Unity. All Sentis models have been validated to work, so you can be sure they will run in Unity.
+
+To get more details about using Sentis, you can read its [documentation](https://docs.unity3d.com/Packages/com.unity.sentis@latest). To get help from others using Sentis, you can ask in its [discussion forum](https://discussions.unity.com/c/ai-beta/sentis)
+
+
+## Types of files
+Each repository will contain several types of files:
+
+* ``sentis`` files: These are the main model files that contain the neural networks that run on Unity.
+* ``ONNX`` files: This is an alternative format which you can include as well as, or instead of, the Sentis files.
+* ``cs`` file: These are C# files which contain the code to run the model on Unity.
+* ``info.json``: This file contains information about the files in the repository.
```

Comment:
~~Not sure if this has already been discussed in the context of downloads tracking, but could it make sense to call this file `config.json`, if still possible? cc @osanseviero~~


Edit: saw the clarification below, it's all good.